resources/js/Pages/Dashboard.jsx

```diff
@@ -0,0 +1,222 @@
+import AuthenticatedLayout from "@/Layouts/AuthenticatedLayout";
+import { Head } from "@inertiajs/react";
+import axios, * as others from "axios";
+import { useState, useRef } from "react";
+import Card from "react-bootstrap/Card";
+import { useMemo, useEffect } from "react";
+import { debounce } from "lodash";
+import { Navbar } from "react-bootstrap";
+import "./../../css/custom.css";
+// import { Container } from "postcss";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const Dashboard = ({ auth, accessToken }) => {
+    const [isLoad, setIsLoad] = useState(false);
+    const [isUniSelected, setIsUniSelected] = useState(false);
+    const [noDataFound, setNoDataFound] = useState(false);
+
+    const timer = null;
+
+    const [arrData, setArrData] = useState([]);
+    const [selectedText, setSelectedText] = useState({});
+
+    const [searchQuery, setSearchQuery] = useState("");
+    const ref = useRef(null);
+
+    const searchInApi = async (event) => {
+        try {
+            setArrData([]);
+            setIsUniSelected(false);
+            setNoDataFound(false);
+            if (event.target.value.length >= 3) {
+                setIsLoad(true);
+
+                let getData = await axios.get(
+                    `http://127.0.0.1/api/search-universities?q=${event.target.value}`,
+                    {
+                        headers: {
+                            Authorization: `Bearer ${accessToken}`,
+                        },
+                    }
+                );
+                setArrData(getData.data.universities);
+                if (getData.data.universities.length == 0) {
+                    setNoDataFound(true);
+                }
+                setIsLoad(false);
+            } else {
+                return true;
+            }
+
+            //hit api from here
```

Comment:
what is this referring to?

Reply:
I added return true, just to be sure if any error or any unexpected behaviour arise it will not throw an exception.